basic/.semaphore.yml

```diff
@@ -20,7 +20,7 @@ blocks:
       jobs:
         - name: npm ci and cache
           commands:
-            - nvm install 12
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to use the latest LTS node version.